packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -404,7 +306,7 @@ const LoggedInPlansFeatureActionButton = ( {
 				showOnMobile={ false }
 				id="downgrade"
 			>
-				<DummyDisabledButton>{ translate( 'Downgrade', { context: 'verb' } ) }</DummyDisabledButton>
+				<DummyDisabledButton>{ text }</DummyDisabledButton>
 				{ isMobile() && (
 					<div className="plan-features-2023-grid__actions-downgrade-context-mobile">
 						{ translate( 'Please contact support to downgrade your plan.' ) }
```

Comment:
This could be one of the "exceptions" perhaps..?

Reply:
Didn't get a chance to take a close look yesterday. I was trying to make sure everything was still functioning properly 😆

Agreed the placeholder / dummy button seems like an exception